### PR TITLE
Add release note about displaying programs on the learner dashboard

### PR DESCRIPTION
**Pull request by rlucioni**
@edx/doc please review.

### CODE REVIEW

File: en_us/release_notes/source/2016/lms/lms_2016-08-15.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.. 
+.. For example:
+.. 
+.. To improve the experience of learners who use screen readers, the
+.. learner dashboard now provides additional, course specific context for
+.. each of the Upgrade to Verified or View XSeries Details options that
+.. appear on this page. (:jira:`ECOM-4269`, :jira:`ECOM-4270`)
+.. 
+
+* The learner dashboard now displays associations to programs of any type.
```
Comment (lamagnifica) [inline en_us/release_notes/source/2016/lms/lms_2016-08-15.rst]: @rlucioni what are "associations"? I think I know what this means, but I'm not sure. 
Maybe,
The courses listed on the learner dashboard now display their associations to any type of program.
?

Comment (lamagnifica) [inline en_us/release_notes/source/2016/lms/lms_2016-08-15.rst]: or, 
Associations to any type of program, and not XSeries programs only, now display for courses on the learner dashboard.

Comment (rlucioni) [inline en_us/release_notes/source/2016/lms/lms_2016-08-15.rst]: @lamagnifica updated.

**Comment by lamagnifica** [comment]
👍

**Pull request merged**